packages/wonder-blocks-icon-button/index.test.js

```diff
@@ -25,3 +44,49 @@ describe("IconButton", () => {
         ).toThrowError("Light is only supported for primary IconButtons");
     });
 });
+
+describe("IconButtonCore", () => {
+    for (const kind of ["primary", "secondary", "tertiary"]) {
```

Comment:
🤷‍♂️ It feels like it'd be nice to have a way to abstract out these permutations, but any solution I can think of would also be a lot of work, so it's probably not worth it!

Reply:
It would be cool to be able generate these tests from flow types.  Maybe when I'm more familiar with react-docgen this will turn out be really easy.